web/packages/teleport/src/DesktopSession/DesktopSession.tsx

```diff
@@ -131,6 +131,7 @@ export function DesktopSession() {
         }
       }}
       aclAttempt={aclAttempt}
+      browserSupportsSharing={navigator.userAgent.includes('Chrome')}
```

Comment:
nit: is it for test that we want to define `browserSupportsSharing` props? Otherwise, we might just define it closer to wherever this is used.

Reply:
This component is used both in Web UI and Connect. In Web UI, sharing works only on Chromium based browsers, in Connect we don't need to verify the user agent as we know it supports sharing.
The side effect is that we can configure it in tests.